### PR TITLE
Show proposal status and add proposal filtering on history page

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/2175
Resolves: https://github.com/daostack/alchemy/issues/190

* adds proposal status for active proposals on history page
* adds proposal filter on history page
* fixes sticky scrolling on history and dao plugin pages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,8 +84,8 @@
     "start-prod": "cross-env NODE_ENV=production SHOW_ALL_DAOS=false NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js  --config webpack.dev.config.js",
     "start-staging-kovan": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=kovan node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
     "start-staging-rinkeby": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=rinkeby node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
+    "start-staging-xdai": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
```
Comment (roienatan) [inline package.json:87]: This change is in order to be consistent with `start-staging-kovan` and `start-staging-rinkeby`?

File: src/components/Dao/DaoHistoryPage.tsx [hunk 2]
```diff
@@ -22,50 +21,126 @@ interface IExternalProps extends RouteComponentProps<any> {
...
           loader=""
-          style={{overflow: "visible"}}
+          style={{ overflow: "visible" }}
           endMessage={null}
         >
-          { proposals.length === 0 ?
-            <span>This DAO hasn&apos;t passed any proposals yet. Checkout the <Link to={"/dao/" + daoState.id + "/proposal/"}>DAO&apos;s installed pluginss</Link> for any open proposals.</span> :
+          {proposals.length === 0 ?
+            this.state.filteredProposalSet ?
+              <span>No proposals found whose title contains the given text.  Note the filter is case-sensitive.</span> :
```
Comment (roienatan) [inline src/components/Dao/DaoHistoryPage.tsx:141]: i18next

File: src/components/Dao/DaoHistoryPage.tsx [hunk 3]
```diff
@@ -22,50 +21,126 @@ interface IExternalProps extends RouteComponentProps<any> {
...
-        <Sticky enabled top={50} innerZ={10000}>
-          <div className={css.daoHistoryHeader}>
-            History
+        <div className={css.paddingTop}>&nbsp;</div>
+
+        <div className={css.topRow}>
+          <div className={css.daoHistoryHeader}>History</div>
+
+          <div className={css.searchBox.concat(`${this.state.filtering ? ` ${css.filtering}` : ""}`)}>
+            <input type="text" name="search" placeholder="Type and press Enter or Tab to filter proposals by title"
```
Comment (roienatan) [inline src/components/Dao/DaoHistoryPage.tsx:124]: i18next

**Review by roienatan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dkent600) [reply package.json:87]: yes, and with alchemy 1.0

File: src/components/Dao/DaoHistoryPage.tsx (hunk 2 above)
Comment (dkent600) [reply src/components/Dao/DaoHistoryPage.tsx]: done

File: src/components/Dao/DaoHistoryPage.tsx (hunk 3 above)
Comment (dkent600) [reply src/components/Dao/DaoHistoryPage.tsx]: done

**Review by roienatan — APPROVED** [review]

**Pull request merged**